sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -20,11 +20,18 @@
           <template #[`item.restrictionCode`]="{ item }">
             {{ item.restrictionCode }}
           </template>
-          <template #[`item.isRestrictionActive`]="{ item }">
-            <status-chip-bypass
-              :is-restriction-active="item.isRestrictionActive"
+          <template #[`item.restrictionStatus`]="{ item }">
+            <status-chip-restriction
+              :status="
+                item.isRestrictionActive
+                  ? RestrictionStatus.Active
+                  : RestrictionStatus.Resolved
+              "
             />
           </template>
+          <template #[`item.bypassStatus`]="{ item }">
+            <status-chip-bypass :is-restriction-active="item.isBypassActive" />
```

Comment:
Minor. Can you please change the attribute name from `is-restriction-active` (isRestrictionActive) to 

`is-bypass-active`?